services/modules/seat/manager.py

```diff
@@ -76,6 +76,25 @@ def delete_user(cls, username):
         return None
 
     @classmethod
+    def disable_user(cls, username):
+        """ Disable user """
+        ret = cls.exec_request('user/{}'.format(username), 'put', active=0)
+        logger.debug(ret)
+        ret = cls.exec_request('user/{}'.format(username), 'put', email="")
```

Comment:
email field always must be filled. SeAT core is using random string in order to seed this field for SSO account, you could do the same, but be sure it's alway unique : ie `sanitizedtimestamp_salt@seat.local`